drivers/uktty/ns16550/Config.uk

```diff
@@ -1,29 +1,12 @@
 menuconfig LIBUKTTY_NS16550
 	bool "NS16550 / 8250"
-	default n
 	depends on ARCH_ARM_64
+	select LIBUKBOOT
+	select LIBUKOFW
 
 if LIBUKTTY_NS16550
+
 menuconfig LIBUKTTY_NS16550_EARLY_CONSOLE
```

Comment:
Since there aren't any submenu configs, we could turn this into `config`. But please ignore this comment if we plan to introduce sub options again with PR #1464 or the PR merging x86 and arm console drivers with NS16550.

Reply:
Good point. I think we should do it correctly here, regardless of how things may evolve in the (near) future.